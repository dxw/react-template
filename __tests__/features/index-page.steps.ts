import { defineFeature, loadFeature } from "jest-cucumber";

import Then from "../helpers/steps/Then";
import When from "../helpers/steps/When";

defineFeature(loadFeature("./index-page.feature"), test => {
  test("Page has content", ({ when, then, and }) => {
    When.iVisitX(when);
    Then.thePageTitleShouldBeX(then);
    Then.iShouldSeeXOnThePage(and);
  });

  test("Page has no accessibility violations", ({ when, then }) => {
    When.iVisitX(when);
    Then.thePageShouldBeAccessible(then);
  });
});
