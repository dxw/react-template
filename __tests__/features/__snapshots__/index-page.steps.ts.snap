// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page Page has content 1`] = `
"<h1>Welcome to dxw's React template!</h1>
"
`;

exports[`Index page Page has no accessibility violations 1`] = `
"<h1>Welcome to dxw's React template!</h1>
"
`;
